src/App.tsx

```diff
@@ -1,14 +1,60 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { FilteredOptions } from './types/FilteredOption';
+import { getTodos } from './api';
+
+function handleFilteredTodos(
+  todos: Todo[],
+  selectedOption: FilteredOptions,
+  query: string,
+) {
+  let filteredTodos = [...todos];
```

Comment:
Why do you need to make a copy, if you use `filter` below, it is not mutating method